CHANGELOG.rst

```diff
@@ -17,6 +17,15 @@ Unreleased
 ~~~~~~~~~~
 
 =======
```

Comment:
this set of ==== shouldn't be in the changelog, I left it in there when I was resolving conflicts